### PR TITLE
Style Guide Improvements

### PR DESCRIPTION
**Pull request by m-ata**

### CODE REVIEW

File: src/views/style-guide/MiscellaneousSection.tsx [hunk 1]
```diff
@@ -14,17 +14,17 @@ const MiscellaneousSection: FunctionComponent<MiscellaneousSectionProps> = ({
 }) => (
   <>
     {styleGuideSections.map(({ frontMatter, source }) => (
-      <Box key={frontMatter.title}>
+      <Box  align-content = "center" key={frontMatter.title}>
         <Heading as="h3" size="lg">
           {frontMatter.title}
         </Heading>
         <Text>{frontMatter.description}</Text>
-        <Box mt={8}>
```
Comment (miedziak) [inline src/views/style-guide/MiscellaneousSection.tsx:22]: todo: we need this `mt` to create some space between the content and the heading -> https://d.pr/R0Ipvm

File: src/views/style-guide/MiscellaneousSection.tsx [hunk 2]
```diff
@@ -14,17 +14,17 @@ const MiscellaneousSection: FunctionComponent<MiscellaneousSectionProps> = ({
 }) => (
   <>
     {styleGuideSections.map(({ frontMatter, source }) => (
-      <Box key={frontMatter.title}>
+      <Box  align-content = "center" key={frontMatter.title}>
```
Comment (miedziak) [inline src/views/style-guide/MiscellaneousSection.tsx:17]: question: is this align-content necessary here ?

File: src/content/style-guide/miscellaneous.mdx [hunk 3]
```diff
@@ -3,9 +3,55 @@ title: 'Miscellaneous'
...
     marginBottom={20}
+    image = "/style-guide/miscellaneous/placeholder-team-member-blue.png"
     title="Team member placeholder [blue]"
     description="Placeholder photo used in team page when no avatar for user is detected"
 />
+
+<StyleGuideResourceCaption
+    marginBottom={20}
+    image = "/style-guide/miscellaneous/logo_caption_dark_background_preview.svg"
+/>
```
Comment (miedziak) [inline src/content/style-guide/miscellaneous.mdx:16]: suggestion: if you include image in this component, I would suggest to name this component little bit different, like ie. "StyleGuideResourceItem"

**Review by miedziak — CHANGES_REQUESTED** [review]

File: src/content/style-guide/miscellaneous.mdx (hunk 3 above)
Comment (m-ata) [reply src/content/style-guide/miscellaneous.mdx]: Nice Idea!

File: src/views/style-guide/MiscellaneousSection.tsx (hunk 2 above)
Comment (m-ata) [reply src/views/style-guide/MiscellaneousSection.tsx]: Not necessary now!

File: src/views/style-guide/MiscellaneousSection.tsx (hunk 1 above)
Comment (m-ata) [reply src/views/style-guide/MiscellaneousSection.tsx:22]: True! In fact, `justifyConenten: space-between` is also not necessary

**Review by miedziak — APPROVED** [review]

**Pull request merged**